packages/cljfmt/package.yaml

```
@@ -0,0 +1,37 @@
+---
+name: cljfmt
+description: A tool for formatting Clojure code
+homepage: https://github.com/weavejester/cljfmt
+licenses:
+  - EPL-1.0
+languages:
+  - Clojure
+  - ClojureScript
+categories:
+  - Formatter
+
+source:
+  # renovate:versioning=loose
+  id: pkg:github/weavejester/cljfmt@0.12.0
+  asset:
+    - target: darwin_arm64
+      file: cljfmt-{{ version }}-darwin-aarch64.tar.gz
+      bin: cljfmt
+    - target: darwin_x64
+      file: cljfmt-{{ version }}-darwin-amd64.tar.gz
+      bin: cljfmt
+    - target: linux_arm64
```

It does however seem like the aarch64 binary is built for GNU/Linux:

```suggestion
    - target: linux_arm64_gnu
```